package.json

```diff
@@ -1,7 +1,48 @@
 {
   "name": "root",
   "private": true,
+  "scripts": {
+    "build": "lerna run build",
```

Comment:
Should this be `npx lerna run build`? It's not otherwise installed.

Reply:
done